app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -100,6 +101,16 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
         }
 
         serviceLocator.intentLiveData.value = Consumable.from(intentData)
+
+        if (BuildConfig.DEBUG) {
+            debugLog.visibility = View.VISIBLE
+            val debugLogStr = "TYPE: ${BuildConfig.BUILD_TYPE} / " +
+                    "FLAVOR: ${BuildConfig.FLAVOR} / VERSION: ${BuildConfig.VERSION_NAME}"
+
+            debugLog.text = debugLogStr
+        } else {
+            debugLog.visibility = View.GONE
```

Comment:
I'm of the opinion that test code shouldn't ship with our APK, so this kind of check worries me a little bit.  Would you be opposed to handling this via source set instead?  We could have a `BuildDisplayer` class with debug and release implementations.  That way the code to set this view up wouldn't even ship to our users.

Reply:
@Baron-Severin  We have test codes in other places (i.e. enable remoteDebugging, show PocketError, etc)